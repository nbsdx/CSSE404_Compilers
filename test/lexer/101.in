// Test of Integers, Delimiters, Comments and Operators
100 + 200;300 - 600;     11 * 22; 55/ 11;;;;;

70 < 80 < 90 < 100 <= 100;;; 100 >= 100 > 90 >
80 >
70 >
60;

5 == 6; 7 == 8; 42 == 42;
42 != 0; 55 != 55;

1 && 1; 0 && 1; 2 || 2; 1 || 0;

!1; !0;
