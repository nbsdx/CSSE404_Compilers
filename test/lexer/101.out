Integer, 100
Operator, +
Delimiter, ;
Integer, 300
Operator, -
Integer, 600
Delimiter, ;
Integer, 11
Operator, *
Integer, 22
Delimiter, ;
Integer, 55
Operator, /
Integer, 11
Delimiter, ;
Delimiter, ;
Delimiter, ;
Delimiter, ;
Delimiter, ;
Integer, 70
Operator, <
Integer, 80
Operator, <
Integer, 90
Operator, <
Integer, 100
Operator, <
Integer, 100
Operator, <=
Delimiter, ;
Delimiter, ;
Delimiter, ;
Integer, 100
Operator, >=
Integer, 100
Operator, >
Integer, 90
Operator, >
Integer, 80
Operator, >
Integer, 70
Operator, >
Integer, 60
Delimiter, ;
Integer, 5
Operator, ==
Integer, 6
Delimiter, ;
Integer, 7
Operator, ==
Integer, 8
Delimiter, ;
Integer, 42
Operator, ==
Integer, 42
Delimiter, ;
Integer, 42
Operator, !=
Integer, 0
Delimiter, ;
Integer, 55
Operator, !=
Integer, 55
Delimiter, ;
Integer, 1
Operator, &&
Integer, 1
Delimiter, ;
Integer, 0
Operator, &&
Integer, 1
Delimiter, ;
Integer, 2
Operator, ||
Integer, 2
Delimiter, ;
Integer, 1
Operator, ||
Integer, 0
Delimiter, ;
Operator, !
Integer, 1
Delimiter, ;
Operator, !
Integer, 0
Delimiter, ;
